spec/integration/schemas/v2/GET_received_text_response.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "GET inbound sms schema response",
+    "type": "object",
+    "title": "GET response v2/inbound_sms",
+    "properties": {
+        "user_number": {"type": "string"},
+        "created_at": {
+            "format": "date-time",
+            "type": "string",
+            "description": "Date+time created at"
+        },
+        "service_id": {"$ref": "definitions.json#/uuid"},
+        "id": {"$ref": "definitions.json#/uuid"},
+        "notify_number": {"type": "string"},
+        "content": {"type": "string"}
+    },
+    "required": [
+        "id", "user_number", "created_at", "service_id",
+        "notify_number", "content"
+    ],
+    "additionalProperties": false
+}
```

Comment:
i don't need you to change these files, but it's probably a good thing to check your sublime settings and ensure that you've got trailing new lines on save turned on. your global sublime config should probably have both of 

```
"ensure_newline_at_eof_on_save": true
"trim_trailing_white_space_on_save": true
```

see https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline